lib/cdo/image_moderation.rb

```diff
@@ -0,0 +1,15 @@
+require 'cdo/azure_content_moderator'
+
+module ImageModeration
+  # Returns a content rating from an external service.
+  # @param [String] image_url a public url to the image to be rated
+  # @return [:everyone|:racy|:adult] Whether the image is suitable for everyone
+  def self.rate_image(image_url)
+    return :everyone unless CDO.azure_content_moderation_key
```

Comment:
As designed, shipping this PR should have no effect until we add the appropriate secrets to the production environment in Chef.